### ISSUE TITLE
Images not loading properly (+massive CPU usage)

### ISSUE THREAD

**Issue by TheBlulitron** [CreativeMD/LittleFrames#74]
Hey there,
I recently booted up my Minecraft server (1.12.2) and found all the images to be just blank white. After enabling VLC in the mod's config, the images loaded again, but my frames dropped to almost 0 FPS. So i updated the mod to the latest version now including WaterMedia. However, while most images work fine again, some of them still appear blank. Even reloading the URL doesn't work. Only if I teleport out of the chunk and then teleport back, it just might load. In certain areas of the map, I also now have massive frame drops since I updated the mod.
I assume the issue is related to VLC and I also found this in the log file (it appears there many times, almost the entire log is actually filled with this exact error over and over). Do you have any idea how to fix this/is it a problem with the mod or just VLC doing things again?

https://pastebin.com/uvhzbVGM

Best regards,
Blulitron

**Comment by SrRapero720** [comment]
you're using a old version because the package is the original package of vlcj and watermedia relocates VLCJ package

**Comment by CreativeMD** [comment]
Sorry, don't see a good way to help out here. Maybe your computer is running out of RAM? Does VLC itself work properly (you can test that outside of minecraft)?

**Comment by TheBlulitron** [comment]
It seems like I found a solution. I updated Xaero's Minimap and Xaero's World Map (which caused issues in the log as well). After that it worked fine again. Yet I'm not sure as of why it caused issues in the first place, since the modpack did work before in it's combination and configuration for months. I assume it had to do with a conflict in overwriting the vanilla world maps or something and VLC might have been an issue. I honestly don't know tho...
